apps/studio.giselles.ai/app/workspaces/[workspaceId]/layout.tsx

```diff
@@ -37,10 +37,10 @@ export default async function Layout({
 				github: gitHubIntegrationState,
 			}}
 			usageLimits={usageLimits}
-			// following settings is test. We'll replace for production.
 			telemetry={{
 				metadata: {
-					example: "metadata",
+					isProPlan: isProPlan(currentTeam),
+					isInternalTeam: isInternalTeam(currentTeam),
```

Comment:
```suggestion
					teamType: currentTeam.type,
```

Reply:
@Rindrics https://github.com/giselles-ai/giselle/pull/496/files#r2011207714
You're right. The part where I commented the suggested change was actually incorrect.
Currently, the plans are only Free and Pro, so keeping it as a boolean might be fine.

That said, I thought that instead of using a boolean, simply having team.type would make the code more adaptable to future changes.